praw/models/reddit/collections.py

```diff
@@ -471,7 +476,10 @@ def create(self, title: str, description: str):
                 "sr_fullname": self.subreddit_fullname,
                 "title": title,
                 "description": description,
-            },
+                "display_layout": display_layout,
+            }
+            if display_layout
+            else {},
```

Comment:
This is not going to work. This will only include the entire data payload if `display_layout` is not `None`.

I was thinking something like this:

```python
data = {
    "sr_fullname": self.subreddit_fullname,
    "title": title,
    "description": description,
}
if display_layout:
    data["display_layout"] = display_layout
return self._reddit.post(
    API_PATH["collection_create"],
    data=data,
)
```

Reply:
Ah, that makes more sense. I didn't really think on this one.

Reply:
This still appears unchanged to me.